common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -377,11 +377,11 @@ def set_last_submission_time(self):
 
     def get_progress(self):
         """
-        For now, just return score / max_score
+        For now, just return weighted earned / weighted possible
         """
-        score_dict = self.get_score()
-        score = score_dict.raw_earned
-        total = score_dict.raw_possible
+        score = self.get_score()
+        earned = score.raw_earned
+        total = score.raw_possible
```

Comment:
Consider {raw,weighted}_{earned,possible} names instead of earned/total and reusing the same names below  for weighted versions.